addons/minio/template/testgrid/k8s-docker.yaml

```diff
@@ -36,3 +36,29 @@
     minio:
       version: "__testver__"
       s3Override: "__testdist__"
+
+- installerSpec:
+    kubernetes:
+      version: "latest"
+    weave:
+      version: "latest"
+    longhorn:
+      version: "latest"
+    containerd:
+      version: "latest"
+    minio:
+      version: "__testver__"
+#      claimSize: "20Gi"
```

Comment:
Is this a comment?  If it is, please add some detail, otherwise get rid of it.

Reply:
it's a "I want to have a test with this param set, but that requires merging first because that param doesn't exist otherwise"

I can add a comment signifying that